common/app/model/meta.scala

```diff
@@ -569,6 +569,8 @@ final case class Tags(
 
   lazy val isClimateChangeSeries = tags.exists(t => t.id =="environment/series/keep-it-in-the-ground")
   lazy val isTheMinuteArticle = tags.exists(t => t.id == "tone/minute")
+  //this is for the immersive header to access this info
+  lazy val isImmersiveAdvertisement = tags.exists( t => t.id == "tone/advertisement-features" )
```

Comment:
Ah ok - the advertisment feature tone gets added automatically when it has a sponsored tag. So I guess making it more generic would be good.